WalletWasabi.Documentation/BackendDeployment.md

```diff
@@ -4,20 +4,23 @@ Consider updating the versions in `WalletWasabi.Helpers.Constants`. If the versi
 
 ```sh
 sudo apt-get update && cd ~/WalletWasabi && git pull && cd ~/WalletWasabi/WalletWasabi.Backend && dotnet restore && cd ~
-sudo service nginx stop
 sudo systemctl stop walletwasabi.service
-sudo killall tor
 bitcoin-cli stop
 sudo apt-get upgrade -y && sudo apt-get autoremove -y
+# in case a new kernel is available:
 sudo reboot
+# service is restarted after boot, so stop it again
+sudo systemctl stop walletwasabi.service
+
 set DOTNET_CLI_TELEMETRY_OPTOUT=1
 bitcoind
 bitcoin-cli getblockchaininfo
-tor
-sudo service nginx start
 dotnet publish ~/WalletWasabi/WalletWasabi.Backend --configuration Release --self-contained false
 sudo systemctl start walletwasabi.service
-pgrep -ilfa tor && pgrep -ilfa bitcoin && pgrep -ilfa wasabi && pgrep -ilfa nginx
+pgrep -ilfa bitcoin
+systemctl status tor@default
+systemctl status walletwasabi
+systemctl status nginx
```

Comment:
This could be a single line 
```suggestion
systemctl status tor@default && systemctl status walletwasabi && systemctl status nginx
```

Reply:
I find it less readable but anyway, you probably want to separate them by `;` to see all outputs, not just until one of them fails.